test/JDBC/input/SELECT_INTO_TEST-vu-verify.sql

```diff
@@ -0,0 +1,399 @@
+
```

Comment:
Just realised that we missed adding tests cases for following scenarios - 
1. When source table's column length is >=64 and/or contains multi-byte characters
2. When source table's column is specified as multi-part, say, `select master.dbo.table.xyz into ...` , `select dbo.table.xyz into ...` , etc